### PR TITLE
Added appium calculator test

### PR DESCRIPTION
**Pull request by lerik14**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -82,6 +74,33 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>pl.pragmatists</groupId>
+            <artifactId>JUnitParams</artifactId>
+            <version>1.1.1</version>
+            <scope>test</scope>
+        </dependency>
+
+
```
Comment (Orkyboy) [inline pom.xml]: empty line

File: src/test/java/pages/CalculatorPage.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    private MobileElement result;
+
+    public CalculatorPage(AndroidDriver<MobileElement> driver) {
+        this.driver = driver;
+        PageFactory.initElements(new AppiumFieldDecorator(driver, Duration.ofSeconds(30)), this);
+    }
+
+    public void performSum(String ... numbers) {
+        for (String number : numbers) {
+            clickOnDigit(number);
```
Comment (Orkyboy) [inline src/test/java/pages/CalculatorPage.java]: I'd rename it to enterNumber

File: src/test/java/pages/CalculatorPage.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            if (number.equals(numbers[numbers.length - 1])) {
```
Comment (Orkyboy) [inline src/test/java/pages/CalculatorPage.java]: that is barely readable, please convert to fori and you'll have access to last elem

File: src/test/java/pages/CalculatorPage.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                equalBtn.click();
+            } else {
+                sumBtn.click();
+            }
+        }
+    }
+
+    public int getResult() {
+        String text = result.getText().replace('−','-');
```
Comment (Orkyboy) [inline src/test/java/pages/CalculatorPage.java:45]: please leave a comment with explanation

File: src/test/java/pages/CalculatorPage.java [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        return Integer.parseInt(text);
+    }
+
+    public void clickOnDigit(String number) {
+        char[] digits = number.toCharArray();
+        for (char digit : digits) {
+            if (digit == '-') {
+                subtractionBtn.click();
+            } else {
+                driver.findElementById("com.google.android.calculator:id/digit_" + digit).click();
```
Comment (Orkyboy) [inline src/test/java/pages/CalculatorPage.java]: Please extract to class variable so you'll have all ids together at the top

File: src/test/java/test/appium_tests/AppiumTest.java [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+import java.net.MalformedURLException;
+import java.net.URL;
+
+@RunWith(JUnitParamsRunner.class)
+public class AppiumTest {
+
+    protected AndroidDriver<MobileElement> driver;
+
+    @Before
+    public void setUp() throws MalformedURLException {
```
Comment (Orkyboy) [inline src/test/java/test/appium_tests/AppiumTest.java:25]: Update readme to clarify which steps need to do to be able to run this test

File: src/test/java/test/appium_tests/AppiumTest.java [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    @After
+    public void shutDown() {
+        driver.quit();
+    }
+
+    @Test
+    @Parameters(method = "sumTestData")
+    public void sumTest(int expectedResult, String ... numbers) throws InterruptedException {
+        CalculatorPage calculatorPage = new CalculatorPage(driver);
+        Thread.sleep(5000);
```
Comment (Orkyboy) [inline src/test/java/test/appium_tests/AppiumTest.java]: reduce to 3s or even check if you can get rid of it

File: src/test/java/test/appium_tests/AppiumTest.java [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+    private Object[] sumTestData() {
+        return new Object[]{
+                new Object[]{150, "123", "5", "22"},
+                new Object[]{-5, "-5", "0"},
+                new Object[]{75, "15", "60"}
+        };
+    }
+
+    @Test
+    public void qwe() {
```
Comment (Orkyboy) [inline src/test/java/test/appium_tests/AppiumTest.java]: qwe qwe 🦆

**Review by Orkyboy — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (lerik14) [reply pom.xml]: deleted

File: src/test/java/pages/CalculatorPage.java (hunk 3 above)
Comment (lerik14) [reply src/test/java/pages/CalculatorPage.java]: done

File: src/test/java/pages/CalculatorPage.java (hunk 4 above)
Comment (lerik14) [reply src/test/java/pages/CalculatorPage.java:45]: left a comment

File: src/test/java/pages/CalculatorPage.java (hunk 2 above)
Comment (lerik14) [reply src/test/java/pages/CalculatorPage.java]: renamed

File: src/test/java/pages/CalculatorPage.java (hunk 5 above)
Comment (lerik14) [reply src/test/java/pages/CalculatorPage.java]: done

File: src/test/java/test/appium_tests/AppiumTest.java (hunk 7 above)
Comment (lerik14) [reply src/test/java/test/appium_tests/AppiumTest.java]: reduced to 3sec

File: src/test/java/test/appium_tests/AppiumTest.java (hunk 6 above)
Comment (lerik14) [reply src/test/java/test/appium_tests/AppiumTest.java:25]: partially done

**Review by Orkyboy — APPROVED** [review]

**Pull request merged**